deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```diff
@@ -87,3 +87,9 @@ spec:
       severity: Error
       summary: "Filesystem has less than 3% inodes left"
       logType: Cluster Configuration
+    - activeBody: |-
+        Your cluster is currently alerting with 'KubeNodeUnschedulableSRE' because a worker node '${node}' can not drain. This is not an actionable alert for the SRE team. Please troubleshoot the node using this reference documentation: https://docs.openshift.com/rosa/rosa_cluster_admin/rosa-cordoning-nodes.
```

Comment:
I don't believe that OCM Agent has the capability of templating variable names the same way our PD notifications do. In this case we'll want to either tell them what metric to look at, or give them a command to run.  See the `Workload preventing machine deletion` notification above.

Additionally - this alert can be triggered just from someone cordoning the node. In this case, I think it might be better to be more vague here as to what's going on.

Maybe something along the lines of "Your cluster currently has at least one unschedulable worker node. Please troubleshoot this node using the following reference documentation: [link]"?

Reply:
(Disregard the template part, I've since been informed that templating is now a thing in OCM Agent). I'd still like to consider a more generic body text, though :D

Reply:
I agree.